tests/test_estimation.py

```diff
@@ -302,6 +302,29 @@ def test_conditional_probability_alive_is_between_0_and_1(self, cdnow_customers)
                 for t in np.arange(recency, 100, 10.):
                     assert 0.0 <= ptf.conditional_probability_alive(freq, recency, t) <= 1.0
 
+    def test_conditional_probability_alive(self, cdnow_customers):
+        """
+        Target taken from page 8,
+        https://cran.r-project.org/web/packages/BTYD/vignettes/BTYD-walkthrough.pdf
```

Comment:
Yeah! I remember a story you offered at Pydata in Seattle about learning that the hard way. I remembered that lesson. :)

I've been reading through the BTYD implementation a good bit as I get up to speed. Maybe I'll find more. :)